lib/internal/checkout.ts

```diff
@@ -41,6 +45,12 @@ export interface CheckoutApi {
     payload: CreateCheckoutPayload,
     subAccountId?: string
   ): Promise<ApiResponse<Checkout>>;
+  completeCheckout(
+    id: string,
+    idempotencyKey: string,
+    payload: CompleteCheckoutPayload,
+    subAccountId: string
```

Comment:
A sub account id is not required. A checkout is already associated with a sub account